Dockerfile

```
@@ -0,0 +1,12 @@
+FROM nginx:alpine
+EXPOSE 80
+ENV LISTEN_PORT=80 \
+  SERVER_URL=otrecorder \
```

This is not the complete URL but the  host - so let's rename this to

```suggestion
  SERVER_HOST=otrecorder \
```